### PR TITLE
Updated helm charts, added sentry injector support

### PR DESCRIPTION
**Pull request by yaron2**
This PR does the following:

1. Update Helm Charts for [Helm 3.0 CRD support](https://helm.sh/docs/topics/chart_best_practices/custom_resource_definitions/) (putting CRDs in a special `/crds` dir)

2. Removed redundant Role and RoleBindings for `sentry`

3. Have the sidecar injector put the trust root as an env var to the pod if mTLS is enabled, similar to how [Linkerd does it](https://linkerd.io/2/features/automatic-mtls/#how-does-it-work).

4. Add default configuration

### CODE REVIEW

**Comment by yaron2** [comment]
@youngbupark, e2e tests are failing because we added a new k8s object which is a CRD, and Helm 2.0 doesn't support that. (see [here](https://github.com/dapr/dapr/runs/409169613?check_suite_focus=true)

In this PR I fixed by changing the way we treat CRDs to be Helm 3.x compliant, but the tests are run using Helm 2.0.

We need to change the e2e tests to use Helm 3.x..

**Comment by youngbupark** [comment]
> @youngbupark, e2e tests are failing because we added a new k8s object which is a CRD, and Helm 2.0 doesn't support that. (see [here](https://github.com/dapr/dapr/runs/409169613?check_suite_focus=true))
> 
> In this PR I fixed this by changing the way we treat CRDs to be Helm 3.x compliant, but the tests are run using Helm 2.0.
> 
> We need to change the e2e tests to use Helm 3.x..

Pls check this pr - https://github.com/dapr/dapr/pull/965

File: charts/dapr/charts/dapr_config/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Dapr configuration
+name: dapr_config
+version: 0.1.0
```
Comment (youngbupark) [inline charts/dapr/charts/dapr_config/Chart.yaml]: Shouldn't we bump up the chart version?

Comment (yaron2) [reply charts/dapr/charts/dapr_config/Chart.yaml]: Yes. Can you commit directly? I'm a little tied up..

Comment (youngbupark) [reply charts/dapr/charts/dapr_config/Chart.yaml]: Let's do it as a separate pr.

**Comment by yaron2** [comment]
Thanks @youngbupark

**Pull request merged**